src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Truck is working right now!");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck stopped working!");
```

Comment:
Similarly, the message in the `stopWork` method should be more informative. It's good practice to include the class name to clarify which machine has stopped working.